src/release/upgrade.md

```diff
@@ -73,6 +118,15 @@ use the `upgrade` command:
 $ flutter pub upgrade
 ```
 
+To update to the _latest possible version_ of
+all the dependencies listed in the `pubspec.yaml` file,
+and automatically update the constraints in that file,
+use the `upgrade` command with `--major-versions`:
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
Flutter lets you update to the _latest possible version_ of
all dependencies in your `pubspec.yaml` file.
To make this update and update the constraints in that file,
run the `upgrade` command with `--major-versions`:
```

Reply:
"Flutter lets you" implies that we could stop you, which seems like a weird implication since this is just a text file.

Reply:
That would break the symmetry with the previous paragraph, which I think would harm readability, but I've made a similar change to reduce the sentence length; I put "This also automatically update the constraints in the `pubspec.yaml` file." after the command and removed the relevant clause from the paragraph in question.